Formula/libint.rb

```diff
@@ -0,0 +1,17 @@
+class Libint < Formula
+  desc "A library for computing electron repulsion integrals efficiently"
+  homepage "https://github.com/evaleev/libint"
+  url "https://github.com/evaleev/libint/archive/v2.7.2.tar.gz"
+  sha256 "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+  license "GPL-2.0-or-later"
+
+  depends_on "boost"
+  depends_on "mpfr"
+  depends_on "python@.3.11"
+
+  def install
+    system "./configure", *std_configure_args
+    system "make"
+    system "make", "install"
+  end
```

Comment:
we need test for this

Reply:
Yeah I was looking at https://github.com/evaleev/libint/blob/master/tests/hartree-fock/hartree-fock.cc, but it is so long!

@evaleev, do you a short example to test if libint works? 

Something like:
https://github.com/Homebrew/homebrew-core/blob/5d9c4b3f2496856471ab9c1cda318c4ad80f956a/Formula/fftw.rb#L74-L94